client/my-sites/plan-features-2023-grid/hooks/use-highlight-label.ts

```diff
@@ -27,23 +26,24 @@ const useHighlightLabel = ( { planName, flowName, currentSitePlanSlug, selectedP
 	);
 	const isSuggestedPlan =
 		selectedPlan && planLevelsMatch( planName, selectedPlan ) && isAvailableForPurchase;
+	const { intent } = usePlansGridContext();
 
-	if ( flowName && isNewsletterFlow( flowName ) ) {
+	if ( isCurrentPlan ) {
```

Comment:
`Non-blocking / Nit`:

This hook could use a test in `client/my-sites/plan-features-2023-grid/hooks/test`

Reply:
this will get removed next. 
p.s. we will likely see more changes coming through that won't contain tests I'm afraid.